src/util/tubemap.js

```diff
@@ -674,14 +674,23 @@ function placeReads() {
   reads.forEach((read, idx) => {
     read.path.forEach((element, pathIdx) => {
       if (!element.hasOwnProperty("y")) {
+        // previous y value from pathIdx - 1 might not exist yet if that segment is also without node 
+        // use previous y value from last segment with node instead 
+        let previousValidY = null;
+        let lastIndex = pathIdx - 1;
+        while (!previousValidY && lastIndex >= 0) {
```

Comment:
I think testing for truthiness here excludes y=0 without a real justification.

```suggestion
        while (previousValidY === null && lastIndex >= 0) {
```